app/controllers/api/v0/users_controller.rb

```diff
@@ -1,38 +1,40 @@
 class Api::V0::UsersController < ApplicationController
 	def show
 		user = User.find(params[:id])
-		
-		if user
-			render json: UserSerializer.new(user)
-		end
+
+		render json: UserSerializer.new(user)
 	end
 
 	def create
 		user = User.new(user_params)
 		if user.save
 			render json: UserSerializer.new(user), status: :ok
 		else
-			raise ActiveRecord::RecordInvalid
+      render json: user.errors.as_json(full_messages: true), status: 400
 		end
 	end
 
 	def destroy
 		user = User.find(params[:id])
-		user.destroy
-
-		render json: { message: "User #{user.email}, was successfully deleted" }, status: :ok
+		if user.destroy
+			render json: { message: "User #{user.email}, was successfully deleted" }, status: :ok
+		else
+      render json: user.errors.as_json(full_messages: true), status: 422
+		end
```

Comment:
Similar to habits#destroy, I'm going to add a RecordNotDestroyed handler in case the user isn't deleted from the database